lib/dat-worker-pool.rb

```diff
@@ -24,16 +22,18 @@ def initialize(options = nil)
     @do_work_proc = options[:do_work_proc]
     @num_workers  = (options[:num_workers] || DEFAULT_NUM_WORKERS).to_i
     @logger       = options[:logger] || NullLogger.new
+    @queue        = options[:queue] || begin
```

Comment:
@jcredding :water_buffalo: Now that this is multi-line, I'd add an empty line above to space it away and not line up the `=`.  Actually, even better, now just move this below the num workers check below so it won't bother building a default queue it the num workers is bad.